java/src/main/java/ai/rapids/cudf/HostColumnVector.java

```diff
@@ -165,6 +165,7 @@ synchronized int getRefCount() {
    */
   public ColumnVector copyToDevice() {
     if (rows == 0) {
+      // TODO this does not work for nested types!!!
```

Comment:
Should we throw here if it's a nested type?

Reply:
The exception is thrown by the constructor that is being called.  I found this because of a bug in my code that caused an empty column to be copied to the device, which should never happen.